jormungandr/src/fragment/mod.rs

```diff
@@ -1,4 +1,5 @@
 mod entry;
+mod expirations;
```

Comment:
May want to host this module somewhere in `utils`, I think we'll need the same generic data structure elsewhere we do GC.